fem/fe/fe_pos.hpp

```diff
@@ -256,6 +257,62 @@ class H1Pos_WedgeElement : public PositiveFiniteElement
 };
 
 
+/// Arbitrary order H1 elements in 3D utilizing the Bernstein basis on a pyramid
+class H1Pos_PyramidElement : public PositiveFiniteElement, FuentesPyramid
```

Comment:
Can you include a reference for this element?

Reply:
I din't have a reference for this but I'll expand the comment.